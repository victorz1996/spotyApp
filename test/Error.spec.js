import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Bootstrapvue from 'bootstrap-vue'
import error from '../components/Error.vue'

let wrapper
beforeEach(() => {
  Vue.use(Bootstrapvue)
  wrapper = shallowMount(error, {
    propsData: {
      mensaje: 'Error',
    },
  })
})

describe('Error', () => {
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  it('HTMl renderizado structure esperada', () => {
    expect(wrapper.element).toMatchSnapshot()
  })

  it('Probando mensaje de error', () => {
    expect(wrapper.vm.mensaje).toBe('Error')
  })
})
