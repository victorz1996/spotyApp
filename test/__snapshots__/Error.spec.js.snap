// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error HTMl renderizado structure esperada 1`] = `
<div>
  <div
    class="alert alert-danger"
  >
    <h3>
      Error
    </h3>
     
    <p>
      Error
    </p>
  </div>
</div>
`;
